### ISSUE TITLE
UDP command support for dials/encoders on StreamDeck+

### ISSUE THREAD

**Issue by PretenderUK** [bitfocus/companion-module-requests#1011]
Hi everyone.
could any help with updating module for tcp/udp to work correctly with new dials/encoders on streamDeck+?
many thanks in advance.

**Comment by MeestorX** [comment]
Does the module support custom variables? If so, use a custom variable for a value in your OSC string and use the rotaries to increment/decrement that variable.

**Comment by josephdadams** [comment]
Are you talking about a MagicQ module?

**Comment by PretenderUK** [comment]
> Are you talking about a MagicQ module?

Yes, I am. unless there is a standard/generic UDP module I could use - that could work too.
